jobclient/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -795,6 +825,7 @@ public static JSONObject jsonizeJob(Collection<Job> jobs)
             jobBuilder.setCommand(json.getString("command"));
             jobBuilder.setPriority(json.getInt("priority"));
             jobBuilder.setStatus(Status.fromString(json.getString("status")));
+            jobBuilder.disableMeaCulpaRetries(json.getBoolean("disable-mea-culpa-retries"));
```

Comment:
@wyegelwel Why this is `disable-mea-culpa-retries` instead of `disable_mea_culpa_retries`. For example, we have `max_retries` instead of `max-retries`